### DISCUSSION TITLE
deploying

### DISCUSSION THREAD

**Discussion by sergiuprt** [dracula/dracula-ui#101] [Q&A] [answered]
Hey everyone!
I'm kinda noob and I have a question:
let's say I've downloaded the components through the NPM, on my local machine. If I want to deploy the code on a host, do I have to install there the package through the NPM again, or can I simply copy the local files to the host?
(I'm writing a website from scratch, some HTML code in PHP files, hence the questions, because I'm not building anything, it's just a copy/paste kind of deploy)

Thanks

**Comment by Rixcy** [comment]
You can copy the local files to the host, just make sure you include `node_modules` and that an `npm install` has been run prior to copying the files across :)

**Comment by zenorocha** [comment, marked as answer]
You can now use the public npm package which should solve all your deployment issues: https://ui.draculatheme.com/migration
